Source/ARTClientOptions.h

```diff
@@ -237,7 +237,7 @@ NS_ASSUME_NONNULL_BEGIN
  * When `true`, enables idempotent publishing by assigning a unique message ID client-side, allowing the Ably servers to discard automatic publish retries following a failure such as a network fault. The default is `true`.
  * END CANONICAL DOCSTRING
  *
- * BEGIN LEGACY DOCSTRING
+ * BEGIN LEGACY DOCSTRING # useful?
```

Comment:
No mention of possible duplicates (which is important since it's an issue with all messengers I've ever tried)